kubernetes-manifests/config.yaml

```diff
@@ -17,7 +17,7 @@ kind: ConfigMap
 metadata:
   name: environment-config
 data:
-  LOCAL_ROUTING_NUM: "123456789"
+  LOCAL_ROUTING_NUM: "883745000"
```

Comment:
Can you update the local routing num in ui tests as well?
 https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/ad2caf03b510433111c86c83aabe7bf95f2c0e4d/.github/workflows/ui-tests/cypress.json#L49

Reply:
Initially thought the ui tests failed because they were flaky, but looks like `1) cannot reference local routing number` is failing

Reply:
Good catch! I told it to run a bunch of times thinking it was just regular flakiness